app/backend/chat_history/cosmosdb.py

```diff
@@ -34,23 +34,50 @@ async def post_chat_history(auth_claims: Dict[str, Any]):
 
     try:
         request_json = await request.get_json()
-        id = request_json.get("id")
-        answers = request_json.get("answers")
-        title = answers[0][0][:50] + "..." if len(answers[0][0]) > 50 else answers[0][0]
+        session_id = request_json.get("id")
+        message_pairs = request_json.get("answers")
+        first_question = message_pairs[0][0]
+        title = first_question + "..." if len(first_question) > 50 else first_question
         timestamp = int(time.time() * 1000)
 
-        await container.upsert_item(
-            {"id": id, "entra_oid": entra_oid, "title": title, "answers": answers, "timestamp": timestamp}
-        )
+        # Insert the session item:
+        session_item = {
```

Comment:
Should we introduce  a version field in case we want to keep migrating schemas in the  future?

Reply:
Ah that's a clever idea, I can add that!
Note that I did change the container name in the Bicep itself, so that the old ones will stick around, but I like the version field too.

Reply:
Added version field, based off what's in the env variable